.github/workflows/test.yml

```diff
@@ -6,69 +6,45 @@ on:
     types: [opened, synchronize]
     branches-ignore: [staging, production]
 
-env:
-  # Number of parallel jobs for jest tests
-  CHUNKS: 3
 jobs:
-  config:
-    runs-on: ubuntu-latest
-    name: Define matrix parameters
-    outputs:
-      MATRIX: ${{ steps.set-matrix.outputs.MATRIX }}
-      JEST_CHUNKS: ${{ steps.set-matrix.outputs.JEST_CHUNKS }}
-    steps:
-      - name: Set Matrix
-        id: set-matrix
-        uses: actions/github-script@v6
-        with:
-          # Generate matrix array i.e. [0, 1, 2, ...., CHUNKS - 1] for test job
-          script: |
-            core.setOutput('MATRIX', Array.from({ length: Number(process.env.CHUNKS) }, (v, i) => i + 1));
-            core.setOutput('JEST_CHUNKS', Number(process.env.CHUNKS) - 1);
-
-  test:
-    needs: config
+  jest:
```

Comment:
Refactored this workflow to fix some [really strange errors](https://expensify.slack.com/archives/C01GTK53T8Q/p1671217362571039). The result is much simpler, so it's a win-win imo.